referenceframe/input.go

```diff
@@ -27,9 +27,6 @@ func JointPositionsFromInputs(f Frame, inputs []Input) (*pb.JointPositions, erro
 	if len(f.DoF()) != len(inputs) {
 		return nil, NewIncorrectDoFError(len(inputs), len(f.DoF()))
 	}
-	if inputs == nil {
-		return nil, errors.New("inputs cannot be nil")
-	}
```

Comment:
why is `inputs` now allowed to be nil?

Reply:
I think this was overly cautious to begin with.  I can add it back but I noticed it was funny this was being checked after the length check (likely doing nothing). The only way it could reach this line of code and be nil is if the DoF of the frame is 0.  In which case the conversion below should be just returning an empty slice anyway.  

Maybe its too cute trying to take this out.  I'll add it back if you want.